xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -22,13 +22,17 @@ limitations under the License.
 #include <string>
 #include <utility>
```

Comment:
Could we decouple SE changes in a separate PR?

Reply:
I separated the SE changes out to this [pr](https://github.com/openxla/xla/pull/3886). Since the rewriter depends on it, we can review that one first?